mysql-test/suite/sys_vars/r/buffered_error_log_size_basic.result

```diff
@@ -0,0 +1,31 @@
+SET @start_value = @@global.buffered_error_log_size;
+SELECT @start_value;
+@start_value
+0
+SET @@global.buffered_error_log_size = 420;
+SELECT @@global.buffered_error_log_size;
+@@global.buffered_error_log_size
+420
+SET @@global.buffered_error_log_size = 0;
+SELECT @@global.buffered_error_log_size;
+@@global.buffered_error_log_size
+0
+SET @@session.buffered_error_log_size = OFF;
```

Comment:
if you use GLOBAL.buffer_err_log_size=OFF, will it accept?

Reply:
No, it fails with incorrect argument type. I'll change this to 0 too.

Reply:
Fixed.